app/assets/javascripts/discourse/lib/highlight-html.js.es6

```diff
@@ -0,0 +1,88 @@
+function highlight(node, pattern, nodeName, className) {
+  if (
+    ![1, 3].includes(node.nodeType) ||
```

Comment:
Can we use `Node.ELEMENT_NODE` and other enums rather than hard coding integers here? It's much more clear.